pkg/registries/artifactory/artifactory.go

```diff
@@ -34,7 +34,7 @@ func newRegistry(integration *storage.ImageIntegration) (types.Registry, error)
 
 // Test implements a valid Test function for Artifactory
 func (r *registry) Test() error {
-	_, err := r.Client.Repositories()
+	err := r.Client.Ping()
```

Comment:
I think we can simplify this even further by just removing this function completely. That way, we just use the `Test()` function as implemented for generic docker registries: https://github.com/stackrox/stackrox/blob/3.73.1/pkg/registries/docker/docker.go#L208

Note this works because the following indicates that the artifactory registry **is** a docker registry:

```
type registry struct {
	*docker.Registry
}
```

Reply:
Actually, we probably no longer need that struct and we can just do what azure does: https://github.com/stackrox/stackrox/blob/3.73.1/pkg/registries/azure/azure.go

Reply:
I considered that and my thought was the error logging specifically mentioning 'artifactory' would be helpful during troubleshooting, but looking at the other integrations like the one you shared - it appears most do as you suggest.  I'll make that change.